src/App.tsx

```diff
@@ -3,55 +3,83 @@
  * https://github.com/facebook/react-native
  * @flow
  */
-
 import * as React from 'react';
 import {
-  Platform,
-  StyleSheet,
-  Text,
-  View
+    Platform,
+    StyleSheet,
+    Text,
+    View,
+    ToolbarAndroid
 } from 'react-native';
+import Toolbar, {
+    LeftButtonTypes,
+} from './Components/Toolbar'
+import icons, {
+    images
+} from './assets/index';
+
 
 const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' +
-    'Cmd+D or shake for dev menu',
-  android: 'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
+    ios: 'Press Cmd+R to reload,\n' +
+        'Cmd+D or shake for dev menu',
+    android: 'Double tap R on your keyboard to reload,\n' +
+        'Shake or press menu button for dev menu',
 });
 
-export default class App extends React.Component<{}, {}> {
-  render() {
-    return (
-      <View style={styles.container}>
-        <Text style={styles.welcome}>
-          Hello! This is the agritrader app!!
-        </Text>
-        <Text style={styles.instructions}>
-          To get started, edit App.js
-        </Text>
-        <Text style={styles.instructions}>
-          {instructions}
-        </Text>
-      </View>
-    );
-  }
+export default class App extends React.Component < {}, {} > {
+    render() {
+        return ( <
+            Toolbar title = 'Agritrader'
```

Comment:
Is it possible to embed the toolbar component in a view? So we can still add other components to the page

Reply:
I think this should be left for the implementation of the App page. No sense in spending time implementing the page only to do it again later

Reply:
Sorry in advance, Github doesn't seem to like that I responded to comments in a review